test/unit/utils.py

```diff
@@ -58,3 +86,13 @@ def assertRaisesRegexp(expected_exception, expected_regexp,
         else:
             excName = str(expected_exception)
         raise unittest.TestCase.failureException("%s not raised" % excName)
+
+
+def get_random_unicode(length):
+    """Get a random Unicode string."""
+    u_str = u''
+    for _ in range(length):
+        u_str += unichr(random.choice(range(0x0370, 0x0377) +
+                                      range(0x0384, 0x038A)))
+
```

Comment:
Yeah 13 different chars is enough. I mean I just want to have a unicode string, I don"t care whether it"s truly random or contains redundancy.